src/v3/src/components/Form/renderers.tsx

```diff
@@ -201,7 +202,7 @@ export default [
     renderer: InputPassword,
   },
   {
-    tester: ({ options: { inputMeta: { type } = {} as Input } }: FieldElement) => type === 'boolean',
+    tester: ({ options: { inputMeta: { type } = {} as Input } }: FieldElement) => type === 'boolean' || type === 'checkbox',
```

Comment:
Customers can send in `checkbox` as the `Input` type and it is expected to be respected and render as such (tested against v2).